make_book.py

```diff
@@ -205,7 +203,7 @@ def make_bilingual_book(self):
             print(e)
             print("you can resume it next time")
             self.save_progress()
-            exit(0)
+            sys.exit(0)
```

Comment:
why need sys  here?

Reply:
`sys.exit(0)` is considered good to be used in production code for the sys module, if that is the case. Using `exit(0)` should generally be used just within interpreters